integration-tests/kafka/src/test/java/io/quarkus/it/kafka/streams/KafkaStreamsTest.java

```diff
@@ -8,18 +8,16 @@
 
 import javax.inject.Inject;
 
-import org.apache.kafka.clients.consumer.Consumer;
-import org.apache.kafka.clients.consumer.ConsumerConfig;
-import org.apache.kafka.clients.consumer.ConsumerRecord;
-import org.apache.kafka.clients.consumer.ConsumerRecords;
-import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.http.HttpStatus;
+import org.apache.kafka.clients.consumer.*;
```

Comment:
can we avoid the star imports here?

Reply:
Of course, I used to check there is no star in imports, but I missed this one.

Don't hesitate if you see others.